public/jwpltx.js

```diff
@@ -61,23 +68,68 @@ window.jwpltx = window.jwpltx || {};
     sendData('i');
   };
 
+  // Process seek event
+  o.seek = function (offset, duration) {
+    isSeeking = true;
+    // Clear interval in case of a live stream not to update time watched while seeking
+    if (uri.pw === -1) {
+      clearLiveInterval();
+    } else {
+      // We need to rewrite progress of the video when seeking to have a valid ti param
+      lastVp = offset;
```

Comment:
This is going to get you a closer value for total seconds watched, but I think it causes some of the seconds to ultimately be attributed to the wrong quantile, doesn't it?

Reply:
Not really. We store current watched seconds in `timeWatched` (seconds watched before seeking). 
Rewriting `lastVp` helps us to starting counting new seconds after seeking using updated video progress as starting point.

Reply:
Right, but my point is if you have 20s quantiles and you watch 3 seconds, then the user scrolls to 10 seconds into another quantile won't you end up saying that the second quantile had 13 seconds watched when the end is reached?  So it's the right number of seconds but in the wrong spots.

It's probably not a huge issue, but what I would expect is that as soon as a user finishes scrolling we send an event for the quantile and duration they were at when they started to scroll and then start counting a new duration in the new quantile.

Reply:
@dbudzins that is how player flow works now. They also wait for the next quantile to send the `t` event with what was before and after seeking. Ott flow and player flow send events now within approximately the same time.

So if you watched 3 seconds after the quantile started and then seek to 10 of another quantile, we both wait 10 more seconds and send 13 seconds.

Reply:
Maybe it was implemented originally in accordance with the docs:

**t event** - `A user has watched past a threshold of a quantile which is determined based on video length`

So technically a threshold it only after certain playback points.

Reply:
It's a bit ambiguous whether watching part of a threshold section then seeking counts as "watched past", but as long as it's consistent with what the player seems to be doing and we have a solid grasp of it that's more important than semantics.